lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -755,6 +752,15 @@ $(SWIFT_BRIDGING_PCH): $(SWIFT_BRIDGING_HEADER)
 	$(SWIFT_FE) -emit-pch $(SWIFT_FEFLAGS) $< -o $(BUILDDIR)/$@
 endif
 
+# Swift Obj-C header.
+ifneq "$(SWIFT_OBJC_HEADER)" ""
+$(SWIFT_OBJC_HEADER):
```

Comment:
I think this should be

$(SWIFT_OBJC_HEADER): $(SWIFT_SOURCES)

Reply:
Ah okay, I wasn't sure whether sources provided up-front needed to be specified as prerequisites – presumably `$(DYLIB_SWIFT_SOURCES)` should also be specified in that case?

Thanks for the review!

Reply:
I guess you are right. I think we should get rid of DYLIB_* and instead use a BUILD_DYLIB=YES variable. But for now, you are absolutely right!